docker-compose.yml

```diff
@@ -14,25 +14,52 @@ services:
     volumes:
       - postgres-db:/var/lib/postgresql/data
 
+  rabbitmq:
+    container_name: rabbitmq
+    image: rabbitmq:3.13.4-management
+    ports:
```

Comment:
maybe it'd be better to add a health check to rabbit mq?
and extend the `depends_on` field in `api-service` definition?

this way `api-service` won't start until rabbit mq is healthy
https://docs.docker.com/compose/startup-order/